app/scripts/modules/ecs/pipeline/stages/findImageFromTags/findImageFromTagsStage.html

```diff
@@ -0,0 +1,8 @@
+<div ng-controller="ecsFindImageFromTagsStageCtrl as findImageFromTagsCtrl" class="form-horizontal">
+  <stage-config-field label="Full Docker Image path (label or SHA-256)">
+    <input type="text" class="form-control input-sm"
+           ng-model="stage.imageLabelOrSha"/>
+  </stage-config-field>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Maybe put the Example value in a div and use CSS instead?

Reply:
I updated the approach to something more sane :-)